app/static/js/eventKiosk.js

```diff
@@ -0,0 +1,104 @@
+var hitEnter = false;
+
+$(document).keydown(function(e){
+  if (e.key === "Escape") {
+    console.log("get off")
+    $("#fullscreenCheck").prop("checked", false)
+    toggleFullscreen();
+  }
+  else if(e.key === "Enter") {
+      hitEnter = true;
+      submitData();
+      hitEnter = false;
+
+  }
+});
+
+function toggleFullscreen() {

```

Comment:
Did you pull this function from somewhere online? If so, you should credit it in a comment